task09/routes/global.route.ts

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import productRouter from './product.route';
+import cartRouter from './cart.route';
+import authRouter from './auth.route';
+import { verifyTokenMiddleware } from '../auth/middlewares';
+
+const router = express.Router();
+
+router.use('/auth', authRouter);
+router.use(verifyTokenMiddleware);
+router.use('/products', productRouter);
+router.use('/profile/cart', cartRouter);
```

Comment:
I would suggest considering this option as well - [link](https://stackoverflow.com/a/27118077). With the growth of an application, it will be way easier to get confused with the order of middlewares.

Reply:
Fixed